src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -44,27 +44,11 @@ export default class List extends Setup {
         this.logger.info(`Device List command invoked for ${platform}`);
 
         return new Promise<any>((resolve, reject) => {
```

Comment:
We may as well not extend `Setup` anymore either, right?

Reply:
I left it for now thinking that perhaps in the future we might want to rethink the approach. But I can take it out now and in the future we can bring it back if we want to.

Reply:
I'd propose taking it out, just because if we don't end up (re-)using it, somewhere down the road someone's going to come back into the code and not know why it's there in the first place. 🙂 We can just add it back in if we need to come back to it.